docs/view.md

```diff
@@ -4,15 +4,31 @@
 Provides a view to render directly within the component’s element. The view is rendered with the
 component’s [can-component::ViewModel] instance.  `<content/>` elements within the view are replaced by the source elements within the component’s tag.
 
-@option {can-stache.renderer} A [can-stache.renderer] returned by [can-stache]. For example:
+@option {can-stache.renderer} A [can-stache.renderer] returned by [can-stache] or a String that will be passed to [can-stache].
+
+For example:
 
 ```js
-Component( {
+import Component from "can-component";
+import stache from "can-stache";
+
+Component.extend( {
 	tag: "my-tabs",
 	view: stache( "<ul>{{#panels}}<li>{{title}}</li> /* ... */" )
 } );
 ```
 
+…can be written as:
+
+```js
+import Component from "can-component";
+
+Component.extend( {
+	tag: "my-tabs",
+	view: "<ul>{{#panels}}<li>{{title}}</li> /* ... */"
+} );
+```
+
```

Comment:
Same as above, I wanted to improve the signature, but the formatting was really messed up.

Here’s what I ended up with:
<img width="636" alt="screen shot 2018-06-21 at 10 07 09 pm" src="https://user-images.githubusercontent.com/10070176/41758747-8b53aab8-759f-11e8-8b7f-632c2dd17a10.png">